corehq/form_processor/backends/sql/processor.py

```diff
@@ -59,7 +59,8 @@ def is_duplicate(cls, xform, lock):
     @classmethod
     def bulk_save(cls, instance, xforms, cases=None):
         try:
-            XFormInstanceSQL.objects.bulk_create(xforms)
```

Comment:
django's bulk_create sucks. it doesn't set the auto increment key so when you try to resave it, it creates a duplicate. i tried to get rid of it completely but was then running into issues with the primary_key being a string. i think i could get it to work just need to put more time into it